src/alr/alr-commands-printenv.adb

```diff
@@ -89,6 +95,10 @@ package body Alr.Commands.Printenv is
                      Cmd.Cmd_Shell'Access,
                      "", "--wincmd",
                      "Use a Windows CMD shell format for the export");
+      Define_Switch (Config,
+                     Cmd.Last_Build'Access,
+                     "", "--last-build",
+                     "Use last build profiles instead of manifest profiles");
```

Comment:
I don't understand the use for this switch.

Reply:
With the new shared builds the actual paths change depending on the build profile. 

Now, `alr printenv` relies on the info in the manifests, always printing the same info. With this, one could do
```
alr build --profiles=*=validation
alr printenv --last-build
```
and get the paths for validation builds.

Reply:
I would consider this to be the default behavior. We already save the last build profile somewhere, right?

Reply:
Yes, they're saved after every build. That's what's being used to set the profiles here indeed.

We could make `alr printenv` understand the `--profiles` switch too, in case one wants to get a particular environment without forcing a build in advance.